bionic/flow.py

```diff
@@ -1817,12 +1819,62 @@ def core__persistent_cache(
 
     @builder
     @decorators.immediate
-    def core__executor(
+    def core__process_executor(
         core__parallel_execution__enabled,
         core__parallel_execution__worker_count,
     ):
         if not core__parallel_execution__enabled:
             return None
-        return Executor(core__parallel_execution__worker_count)
+        return ProcessExecutor(core__parallel_execution__worker_count)
+
+    builder.assign("core__aip_execution__enabled", False, persist=False)
+    builder.assign("core__aip_execution__gcp_project", None, persist=False)
+    builder.assign("core__aip_execution__docker_image", "bionic:latest", persist=False)
```

Comment:
Is this image publicly accessible, or just within Square?

Reply:
Ah right, this is the image you create using the dockerfile in the base PR. Maybe we should have instructions for people on how to create the docker image.

For reference, these are the commands I use:
```
docker build -t bionic:latest .
docker tag bionic:latest gcr.io/sq-mlf-dev/bionic:latest
docker push gcr.io/sq-mlf-dev/bionic:latest
```